core/src/scss/utilities/variables/components/_nav.scss

```
@@ -0,0 +1,168 @@
+) !default;
+
+//
+// $nav-dark-colors
+//
+// Color map for the desktop version of the Main Nav dark variant 1st level items.
+//
+// Markup: $nav-dark-colors: (
+//  h-link:                               $color-white,
+//  h-link--hover:                        $color-white,
+//  h-border--hover:                      $color-white,
+//  h-link--active:                       $color-white,
+//  h-border--active:                     $color-vivid-red,
+//  h-link--active-hover:                 $color-white,
+//  h-border--active-hover:               $color-white,
+//  h-link--expanded:                     $color-white,
+//  h-link--expanded-hover:               $color-white,
+//  h-border--expanded:                   $color-white,
+//  h-border--expanded-hover:             $color-vivid-red,
+//  h-border--expanded-active:            $color-white,
+//) !default;
+//
+// Styleguide: Variables.Components.nav-colors.variant-dark
+//
+
+$nav-dark-colors: (
+    h-link:                               $color-white,
+    h-link--hover:                        $color-white,
+    h-link--active:                       $color-white,
+    h-link--active-hover:                 $color-white,
+    h-link--expanded:                     $color-white,
+    h-link--expanded-hover:               $color-white,
+
+    h-border--hover:                      $color-white,
+    h-border--active:                     $color-vivid-red,
+    h-border--active-hover:               $color-white,
+    h-border--expanded:                   $color-white,
+    h-border--expanded-hover:             $color-vivid-red,
+    h-border--expanded-active:            $color-white,
+) !default;
+
+//
+// $nav-light-colors
+//
+// Color map for the Main Nav and Secondary Nav light variants.
+//
+// Markup: $nav-light-colors: (
+//  bg:                         $color-white,
+//  divider:                    #d9d9d9,
+//  link:                       $color-black,
+//  link--hover:                 $color-black,
+//  border--hover:               $color-black,
+//  link--active:                $color-bright-red,
+//  border--active:              $color-bright-red,
+//  link--expanded:              $color-black,
+//  border--expanded:            transparent,
+//  border--active-expanded:     $color-black,
+//  h-link:                               $color-black,
+//  h-link--hover:                        $color-bright-red,
+//  h-border--hover:                      $color-bright-red,
+//  h-link--active:                       $color-black,
+//  h-border--active:                     $color-black,
+//  h-border--current:                    $color-driftwood,
+//  h-link--active-hover:                 $color-bright-red,
+//  h-border--active-hover:               $color-bright-red,
+//  h-link--expanded:                     $color-black,
+//  h-border--expanded:                   $color-driftwood,
+//) !default;
+//
+// Styleguide: Variables.Components.nav-colors.variant-light
+//
+
+$nav-light-colors: (
```

I would prefer a name more reminiscent of BEM naming:
```suggestion
$nav-colors--light: (
```